ci/publish-local.sh

```diff
@@ -2,6 +2,6 @@
 
 set -eux
 
-mill -i all __.publishLocal release
+./mill -i all __.publishLocal executable
 
-mv out/release/dest/mill ~/mill-release
+mv out/executable/dest/mill ~/mill-release
```

Comment:
Is there some reason why this is a `mv` and not a `cp`? Currently it violates the policy to not manually change everthing under `./out`. A sub-sequent run of `mill executable` will just skip because it is cached, although there is no executable anymore.

Reply:
@lefou no reason, should probably be a `cp`

Reply:
Good to know, thanks.